src/lib/edge-functions/proxy.mjs

```diff
@@ -110,7 +109,7 @@ export const initializeProxy = async ({
   const userFunctionsPath = config.build.edge_functions
   const isolatePort = await getAvailablePort()
   const buildFeatureFlags = {
-    edge_functions_npm_modules: isFeatureFlagEnabled('edge_functions_npm_modules', siteInfo),
+    edge_functions_npm_modules: true,
```

Comment:
I think there's another place where we need to set this, so it applies to `ntl build`